woof-code/rootfs-skeleton/root/.xinitrc

```diff
@@ -52,6 +52,12 @@ if [ "$CURRENTWM" = "startkde" ];then
  /sbin/pup_event_frontend_d & #130525
  exec startkde
 fi
+CURRENTWM="`cat /etc/windowmanager`"
+if [ "$CURRENTWM" = "startxfce4" ];then
+ /sbin/pup_event_frontend_d & #130525
+ exec startxfce4
+rox --pinboard= 
```

Comment:
This line is never executed

Reply:
ha , you're right ...  can you remove the rox line? or do you want another pull request